### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by a268-Beefy** [Tojaso/Buffle#1]
Ok pal i am gonna be your first =D

As soon comments are closed and i would like to post some revievs ill put them here, but as for bug this 1 happence when i go to ESC > Interface > Addons >Bufflehead (it is empty btw, there is noting but the named tab)

1x ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-79.lua:1956: Bufflehead Options isn't registed with AceConfigRegistry, unable to open config
[string "=[C]"]: ?
[string "@Masque\Libs\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-79.lua"]:1861: in function `Open'
[string "@Masque\Libs\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-79.lua"]:1956: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1954>
[string "=[C]"]: ?
[string "@Masque\Libs\AceGUI-3.0\AceGUI-3.0-41.lua"]:72: in function <Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua:70>
[string "@Masque\Libs\AceGUI-3.0\AceGUI-3.0-41.lua"]:306: in function `Fire'
[string "@Masque\Libs\AceGUI-3.0-41\widgets\AceGUIContainer-BlizOptionsGroup.lua"]:20: in function <...GUI-3.0\widgets\AceGUIContainer-BlizOptionsGroup.lua:19>
[string "=[C]"]: in function `Show'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:29: in function `InterfaceOptionsList_DisplayPanel'
[string "@FrameXML\InterfaceOptionsFrame.lua"]:47: in function `InterfaceOptionsListButton_OnClick'
[string "*:OnClick"]:2: in function <[string "*:OnClick"]:1>

As for reviev:
First of all i wished to say YAY IT IS WORKING! and yet i cant say that this project is well done, it is more like it is fresh and raw. It would be prefect to see better discription about addon on Curse page, same as a list of short commands that open config window. Using this addon I noticed that there is a huge lack of such options as target buffs or debuffs, in the end it is a rather strange decision to use a separate addon for each unit (player/target/tot/focus/etc). Would be nice to see same-debuff stacking option in this project.
After several years of using Raven, it's really nice to see how you've organized Tabs with the buff/debuff string's settings, gj.
And I think the cherry on the cake of your addon can be well-developed combat-short-buff/debuff bars, with black&white list system, that can be used near Unit frames to show crucial buffs and debuffs during combat and filter off rest useless crap.
Thank you for your work man.

**Comment by Tojaso** [comment]
First of all, I am amazed that anyone has downloaded and tested this prior to its first official release. Thank you for taking the time!

I will check out the error with AceConfigRegistry--it probably is because I just changed options to load-on-demand.

I just got the build process working with curseforge yesterday and started working on the description text there today. Hopefully it clarifies the relatively limited scope of this addon. The description is a work-in-progress and will be expanded further, plus images will be added.

Using Blizzard's secure execution framework (to meet primary goal of right-click cancel in combat) completely rules out filtering either by duration or black/white list. Of course, I expect people wanting those features will use Raven or something similar too and only use Bufflehead for cancellable player buffs.

Another major goal for Bufflehead is simplified options panel so I am glad you like the tab organization. This is also why I haven't address two other potential features: (1) separate settings for buffs and debuffs, (2) additional units given that the Blizzard support might work for target/focus/etc. (although I have never seen an example of anyone using them for other than player buffs/debuffs). Maybe in some future release...

Again, thank you for taking a look. Your comments will lead to more tests, text edits, and options panel changes. Bufflehead should look a lot more polished in another week. I am currently reworking the Setup process around templates so a lot is going to change soon.
